File: 1. Model/Brincadeira.py


class Brincadeira:

    def __init__(self, nome, qtdCriancas, descricao):
        self.nome        = nome
        self.qtdCriancas = qtdCriancas
        self.descricao   = descricao
        





    #  - - - - - - - - - - GETS & SETS - - - - - - - - - -
    ##GETS
    def getNome(self):
        return self.nome

    def getQtdCriancas(self):
        return self.qtdCriancas

    def getDescricao(self):
        return self.descricao


    ##SETS
    def setNome(self, n):
        self.nome = n

    def setQtdCriancas(self, qc):
        self.qtdCriancas = qc

    def setDescricao(self, d):
        self.descricao = d

    # - - - - - - - - - - //// /  //// - - - - - - - - - -

File: model/Brincadeira.java
package model;



public class Brincadeira {

	private String nome, descricao;
	private int    qtdCriancas;



	public Brincadeira(String n, String d, int qc){

		this.nome 	     = n;
		this.descricao   = d;
		this.qtdCriancas = qc;
	}



	// Gets
	public String getNome(){
		return this.nome;
	}

	public String getDescricao(){
		return this.descricao;
	}

    public int getQtdCriancas(){
    	return this.qtdCriancas;
    }


    // Sets
    public void setNome(String n){
    	this.nome = n;
    }
    public void setQtdCriancas(int q){
    	this.qtdCriancas = q;
    }
    public void setDescricao(String d){
    	this.descricao = d;
    }
}
